docs/user-guide/cli-install-configure-zosmf-security.md

```diff
@@ -1,21 +1,27 @@
 # Configuring z/OSMF Security
 
-Review the tasks that security administrators must complete to configure z/OSMF security for your Zowe CLI implementation.
+Review the tasks that security administrators must complete to configure z/OSMF security for your implementation of Zowe client-side components. **(Is "Zowe client-side components" correct?)**
 
 :::info Required role: security administrator
 :::
 
 ## Configuring z/OS REST services SAF security
 
-A security administrator must configure security to allow z/OSMF System Authorization Facility (SAF) access to the resources that Zowe CLI requires. Zowe CLI uses REST endpoints that are associated with each z/OSMF REST API. After you complete all z/OSMF and z/OSMF cloud provisioning configurations, you can test your connection to z/OSMF to verify that Zowe CLI can communicate with z/OS systems.
+:::note
+
+If you are connecting to the mainframe with other methods than a z/OSMF profile, you do not need to configure z/OSMF security. Other connection options might include using FTP, or your custom API.
```

Comment:
@adam-wolfe @JillieBeanSim -- is this note accurate/sufficient re: other mainframe connections besides z/osmf profile?

Reply:
I would suggest rearranging to: "If you are connecting to the mainframe with **methods other** than a z/OSMF profile, ..."

Other than that, it looks OK.